v2/client.go

```diff
@@ -744,6 +757,24 @@ func WriteBufferSize(writeBufferSize int) ClientOption {
 	}
 }
 
+// SetConnectionToLowLatencyServer is a ClientOption to set the client to connect
+// to server with lowest latency.
+func SetConnectionToLowLatencyServer() ClientOption {
+	return func(o *ClientOptions) error {
+		o.ServerSelection = Latency
+		return nil
+	}
+}
+
+// SetConnectToLowWorkLoadServer is a ClientOption to set the client to connect
+// to server with lowest work loqd.
+func SetConnectToLowWorkLoadServer() ClientOption {
```

Comment:
```suggestion
func SetConnectToLowWorkloadServer() ClientOption {
```

To be consistent with naming of `Workload` SelectionCriteria.